ptf/saiswitch.py

```diff
@@ -100,7 +287,6 @@ def setUp(self):
 
     def runTest(self):
         self.availableIPv4RouteEntryTest()
```

Comment:
Can we remove this too since you have duplicated it above?

Reply:
removed.

then, what's your opinion about smaller those cases or use the PTF structure to call them in methods but not class level? or, we can use both of them in different scenarios?

there is the code for refactor in PTF the https://github.com/p4lang/ptf/pull/168

Reply:
I think both in different scenarios is a good way forward. There are some tests which have multiple methods but do very little in each method. Splitting them into multiple classes is not worth our time.